apps/system-e2e/src/integration/air-discount-scheme/homepage.spec.ts

```diff
@@ -0,0 +1,58 @@
+// import { idsLogin } from '../../support/commands'
+import {
+  aliasQuery,
+  getFakeUser,
+  getFamily,
+  getBaseUrl,
+  getDiscountData,
+} from '../../support/utils'
+
+describe('Home page', () => {
+  const baseUrl = getBaseUrl(Cypress.config())
+
+  const fakeUser = getFakeUser(Cypress.env('fakeUsers'), 'gervimaður afríka')
+  const family = getFamily(fakeUser)
+
+  beforeEach(() => {
+    cy.idsLogin({
+      phoneNumber: fakeUser.phoneNumber,
+      urlPath: '/min-rettindi',
+    })
+    cy.intercept('POST', `${baseUrl}/api/graphql`, (req) => {
+      aliasQuery(req, 'DiscountsQuery')
+    })
+  })
+
+  it(`should return discounts response array`, () => {
+    cy.visit('/min-rettindi')
+    cy.wait('@DiscountsQuery', { timeout: 20000 }).then((data) => {
+      assert.isNotNull(
+        data.response?.body.data.discounts,
+        'Discounts response exists',
+      )
+    })
+  })
+
+  it(`should have user ${fakeUser.name} with valid discount code`, () => {
```

Comment:
Specifying the name of the user is not necessary IMO in the test case title
```suggestion
  it(`user should have a valid discount code`, () => {
```

Reply:
since we are having stability issues with gervimenn and testing out different ones, I think this visual reference helps with knowing who is being tested